src/xcode/ENA/ENA/Source/HTTPClient/Service/Services/_Service+Default.swift

```diff
@@ -157,11 +157,11 @@ extension Service {
 		return nil
 	}
 
-	func hasNoStatusCodeCachePolicy<R>(
+	func hasCachePolicyStatusCode<R>(
```

Comment:
I think old word order made more sense: `hasStatusCodeCachePolicy` because it returns if it has a certain cache policy and not if it has a certain status code.